Tests/UtilityBeltDataTests/Protocols/CoreDataOperatorTesting.swift

```diff
@@ -7,68 +7,284 @@ import XCTest
 
 protocol CoreDataOperatorTesting: XCTestCase {
     var coreDataOperator: CoreDataOperator { get set }
+    
+    func testNewInstance()
+    
+    func testExists()
+    func testExistsWithPredicate()
+    
+    func testCount()
+    func testCountWithPredicate()
+    
+    func testDeleteSingleObject()
+    func testDeleteAllObjects()
+    func testDeleteAllObjectsWithPredicate()
 }
 
 extension CoreDataOperatorTesting {
-    func count(file: StaticString = #file, line: UInt = #line) {
-        self.measureAndCatch {
-            let userCount = try self.coreDataOperator.count(of: User.self)
+    // MARK: Create
+    
+    func verifyNewInstanceSucceeds(file: StaticString = #file, line: UInt = #line) {
+        do {
```

Comment:
There shouldn't be any reason to wrap this in a `do/catch` block. I am assuming you did this to get the `file` and `line` to work, but all we're calling `try` on in this function is `XCTUnwrap`, so this can just be marked `throws` and it'll get everything right I believe.